src/Symfony/Component/Serializer/composer.json

```diff
@@ -24,9 +24,10 @@
         "symfony/property-access": "~2.8|~3.0",
         "symfony/http-foundation": "~2.8|~3.0",
         "symfony/cache": "~3.1",
-        "symfony/property-info": "~2.8|~3.0",
+        "symfony/property-info": "~3.1",
         "doctrine/annotations": "~1.0",
-        "doctrine/cache": "~1.0"
+        "doctrine/cache": "~1.0",
+        "phpdocumentor/reflection-docblock": "~3.0"
```

Comment:
Because I use the `ReflectionExtractor` in a test (and it's a soft dependency of PropertyInfo).

Theoretically, it should also work with PropertyInfo 2.8 (it's why I've not enforced `<3.1` in a `conflict` section). But 2.8 requires `phpdocumentor/reflection:^1.0.7` and 2.1+`phpdocumentor/reflection-docblock:^3.0`. And those 2 dependencies are in conflict...

So I just required PropertyInfo `^3.1` for the sake of simplicity.